x-pack/platform/plugins/shared/fleet/server/routes/epm/index.ts

```diff
@@ -91,8 +92,19 @@ export const INSTALL_PACKAGES_AUTHZ: FleetAuthzRouteConfig['fleetAuthz'] = {
   integrations: { installPackages: true },
 };
 
-export const READ_PACKAGE_INFO_AUTHZ: FleetAuthzRouteConfig['fleetAuthz'] = {
-  integrations: { readPackageInfo: true },
+export const INSTALL_PACKAGES_SECURITY: RouteSecurity = {
+  authz: {
+    requiredPrivileges: [
+      FLEET_API_PRIVILEGES.INTEGRATIONS.ALL,
+      FLEET_API_PRIVILEGES.AGENT_POLICIES.ALL,
+    ],
+  },
+};
+
+export const READ_PACKAGE_INFO_SECURITY: RouteSecurity = {
+  authz: {
+    requiredPrivileges: [FLEET_API_PRIVILEGES.INTEGRATIONS.READ],
```

Comment:
should it be any of these? https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/fleet/common/authz.ts#L154